src/pages/en/guides/baas/appwriteio.mdx

```diff
@@ -0,0 +1,777 @@
+---
+title: Appwrite & Astro
+description: Add authentication, storage, and realtime data to your Astro application
+layout: ~/layouts/BackendAsAServiceLayout.astro
+service: Appwrite
+stub: false
+---
+import FileTree from '~/components/FileTree.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+
+[Appwrite](https://appwrite.io/) is a self-hosted backend-as-a-service platform that provides developers with all the core APIs required to build any application.
+
+## Integrating with Astro
+
+In this section, we'll use the [Appwrite JavaScript SDK](https://appwrite.io/docs/getting-started-for-web#getSDK) to connect your Appwrite project to Astro.
+
+### Prerequisites
+
+To get started, you will need to have the following:
+
+1. **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
+
+2. **An Appwrite instance, or hosted instance**. If you don't have an account, you can [sign up](https://cloud.appwrite.io/login) for a free account and create a new Appwrite project. You can also use an existing project if you have one. 
```

Comment:
Let's make it clear which type of project they need to create; the creation flow has multiple options.

Also, what value do we use for `hostname`?